charts/shoot-core/components/charts/nodelocal-dns/templates/daemonset.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: node-local-dns
+  namespace: kube-system
+  labels:
+    gardener.cloud/role: system-component
+    garden.sapcloud.io/role: system-component
+    origin: gardener
+    k8s-app: node-local-dns
+spec:
+  updateStrategy:
+    rollingUpdate:
+      maxUnavailable: 10%
+  selector:
+    matchLabels:
+      k8s-app: node-local-dns
+  template:
+    metadata:
+      labels:
+        k8s-app: node-local-dns
+        networking.gardener.cloud/to-dns: allowed
+      annotations:
```

Comment:
Can you add this same comment here?
https://github.com/gardener/gardener/blob/master/charts/shoot-core/components/charts/coredns/templates/coredns-deployment.yaml#L27